docs/serving/autoscaling/scale-bounds.md

```diff
@@ -217,6 +213,59 @@ delay period.
           scale-down-delay: "15m"
     ```
 
+## Stable window
+
+The stable window defines minimal time that passes between decisions to change the scale of a revision. [Panic mode](kpa-specific.md) can override this setting.
```

Comment:
I think this isn't quite right. We average over the stable and panic windows to get the _input_ for decisions, but we actually make decisions [every two seconds by default](https://github.com/knative/serving/blob/61178b2e8bf8d8afe870151ff59ee9724739b496/pkg/autoscaler/scaling/multiscaler.go#L36) . I'd say something like:

```suggestion
The stable window defines the sliding time window over which metrics are averaged to provide the input for scaling decisions when the autoscaler is not in [Panic mode](kpa-specific.md).
```